src/main/proto/ga4gh/common.proto

```diff
@@ -5,6 +5,11 @@ syntax = "proto3";
 
 package ga4gh;
 
+message GAException {
```

Comment:
Please comment as to how this is meant to be used (or at least port the comments from before):

```
/**
A general exception type.
*/
error GAException {
  /** The error message */
  string message;

  /** The numerical error code */
  int errorCode = -1;
}
```

My preference would probably be:

```
// When returning an HTTP error response, a server may also return a JSON formatted GAException
// to better describe the error.
message GAException {
  // Numerical error code (should match the HTTP response code).
  int32 error_code = 1;

  // The error message.
  string message = 2;
}
```